### PR TITLE
Single server entrypoint

### PR DESCRIPTION
**Pull request by matveyvarg**

### CODE REVIEW

File: deker_server_adapters/factory.py [hunk 1]
```diff
@@ -125,12 +124,18 @@ def get_collection_adapter(
         """
         return ServerCollectionAdapter(self.ctx)
 
-    def do_healthcheck(self, ctx: CTX, in_cluster: bool = False) -> Optional[Dict]:
+    def __uri_has_servers(self, ctx: CTX) -> bool:
+        """Check if URI has servers.
+
+        :param ctx: Context with URI
+        """
+        return hasattr(ctx.uri, "servers") and ctx.uri.servers
```
Comment (SerGeRybakov) [inline deker_server_adapters/factory.py:132]: Do we really need this method? Can't we just check `if ctx.uri.servers`?  
As I see, deker.uri always has this attribute (may be we shall refactor it a bit).

File: deker_server_adapters/utils.py [hunk 2]
```diff
@@ -26,8 +28,9 @@ def _request(url: str, node: str, client: Client, request_kwargs: Optional[Dict]
             response = client.get(request_url, **request_kwargs)
         else:
             response = client.get(request_url)
-    except Exception:
-        logger.exception(f"Coudn't get response from {node}")  # noqa
+    except Exception as e:
+        traceback.print_exc()
```
Comment (SerGeRybakov) [inline deker_server_adapters/utils.py:32]: May be we want to print just the last exception from trace (`traceback.print_exc(-1)`)?  
It's too loud now.

File: deker_server_adapters/factory.py (hunk 1 above)
Comment (matveyvarg) [reply deker_server_adapters/factory.py:132]: I'm not sure tbh that `deker.uri` would have `servers` attribute always, so it's just a precaution. Might be refactored later.

Comment (SerGeRybakov) [reply deker_server_adapters/factory.py:132]: Yes, it does.
image

File: deker_server_adapters/utils.py (hunk 2 above)
Comment (SerGeRybakov) [reply deker_server_adapters/utils.py]: https://github.com/openweathermap/deker-server-adapters/pull/23/commits/23e745a77945bccefa6b7323b89a34652356450e

File: deker_server_adapters/factory.py (hunk 1 above)
Comment (SerGeRybakov) [reply deker_server_adapters/factory.py]: https://github.com/openweathermap/deker-server-adapters/pull/23/commits/dff1e28bf33a2f7e21d2ff8aef5e4ed161e36c21

**Review by SerGeRybakov — APPROVED** [review]

**Pull request merged**